helm/examples/nfs/hpcc-nfs/values.schema.json

```diff
@@ -30,17 +41,42 @@
   "title": "Values",
   "type": "object",
   "definitions": {
-    "storage": {
+    "plane": {
       "type": "object",
-      "required": [ "size" ],
       "properties": {
-        "size": {
+        "name": {
+          "description": "Prefix for the pvc name",
           "type": "string"
         },
-        "sku": {
-          "type": "string"
-        }
+	"subPath": {
+	  "description": "sub path from common.hostpath on host where the pvc is created",
+	  "type": "string"
+	},
+        "size": {
```

Comment:
indendation issue, tab used instead of spaces?

Reply:
fixed

Reply:
Double checked. It was fixed.